kazoo/handlers/eventlet.py

```diff
@@ -13,6 +13,7 @@
 from eventlet.green import threading as green_threading
 from eventlet import queue as green_queue
 
+from kazoo.exceptions import KazooTimeoutError, TimeoutError
```

Comment:
Another way to suppress the lint is adding a `__all__ = [..., 'TimeoutError']` line.